.travis.yml

```diff
@@ -161,6 +161,17 @@ jobs:
       env: TAV=knex,ws,graphql,express-graphql
       script: tav --quiet
 
+    # Node.js 0.10
+    - stage: 'test'
+      script: make docs
```

Comment:
This should be `npm run docs` instead (interesting that `make docs` exists with exit code zero when there's no Makefile!)